src/main/resources/assets/fabric-fluid-volume-v1/lang/en_us.json

```diff
@@ -0,0 +1,6 @@
+{
+  "fabric.fluid.and": " and ",
```

Comment:
Maybe not use `fabric` as part of the translation key.

But at the same time, if this is a part of the api, the fabric key in text would make sense